### PR TITLE
[GEOS-10328] Expire completed and stale importer contexts

### PR DESCRIPTION
**Pull request by aaime**
[GEOS-10328](https://osgeo-org.atlassian.net/browse/GEOS-10328)

See ticket

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [x] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [x] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [x] New unit tests have been added covering the changes.
- [x] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [x] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [x] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [x] Bug fixes and small new features are presented as a single commit.
- [x] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

**Review by dromagnoli — APPROVED** [review]
LGTM

File: src/community/importer-jdbc/src/main/java/org/geoserver/importer/jdbc/JDBCImportStore.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import org.geoserver.importer.ImportContext;
-import org.geoserver.importer.ImportStore;
+import org.geoserver.importer.ImportStore;
```
[the remark this answers is not in the document]
Comment (aaime) [reply src/community/importer-jdbc/src/main/java/org/geoserver/importer/jdbc/JDBCImportStore.java]: The Google Java formatter is actually run on community modules too (it complained, I fixed it).

File: data/release/global.xml [hunk 2]
```diff
@@ -15,7 +15,20 @@
     <onlineResource>http://geoserver.org</onlineResource>
     <verbose>false</verbose>
     <verboseExceptions>false</verboseExceptions>
+    <metadata>
+      <map>
+        <entry>
+          <string>CogSettings.Key</string>
```
Comment (jodygarnett) [inline data/release/global.xml:21]: @aaime did you intend to include COG settings as part of this change?

Comment (aaime) [reply data/release/global.xml:21]: I did not.

Comment (aaime) [reply data/release/global.xml:21]: However looking at the diff, I don't see a change in globa.xml?
The current global.xml does not contain that metadata entry either:

https://github.com/geoserver/geoserver/blob/main/data/release/global.xml

Not sure why you're pointing at this particular PR?

Comment (aaime) [reply data/release/global.xml:21]: Ah, if you look at the history, the change has been reverted in the same day. Again, what brought you to look at this particular PR?

**Pull request merged**